validator/keymanager/v2/direct/direct.go

```diff
@@ -247,6 +248,48 @@ func (dr *Keymanager) CreateAccount(ctx context.Context) (string, error) {
 	return accountName, nil
 }
 
+// DeleteAccount takes in a public key and removes the account entirely. This includes its disk keystore and cached keystore.
+func (dr *Keymanager) DeleteAccount(ctx context.Context, publicKey []byte) error {
```

Comment:
Can we consolidate the deletion of multiple accounts into this one function? We can take in a slice of pubkeys instead